src/components/Menu/Menu.vue

```diff
@@ -33,7 +33,15 @@
         :href="item.item.url"
         :target="item.item.target"
       >
-        <span class="uic-menu-item-child">
+        <span
+          :class="[
+            'uic-menu-item-child',
+            {
+              '!text-gray-200 !hover:bg-none !cursor-not-allowed':
```

Comment:
Learning time for me: Why do you add `!` in all classes here?

Reply:
Adding a `!` to tailwind classes adds important rules to their styles. Primevue's more opinionated components are a bit stubborn about style overrides, so this is a workaround for a flow that (I believe) we can't achieve through their standard passthrough flows since they don't have a disabled styles class natively for their menu items.

Reply:
thanks... !important is not my favorite css feature.